cnf-tests/pod-utils/cyclictest-runner/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"k8s.io/klog"
+	"os/exec"
+	"strconv"
+	"time"
+
+	podutils "github.com/openshift-kni/cnf-features-deploy/cnf-tests/pod-utils"
+)
+
+const cyclictestBinary = "/usr/bin/cyclictest"
+
+func main() {
+	klog.InitFlags(nil)
+
+	var rtPriority = flag.String("rt-priority", "1", "specify the SCHED_FIFO priority (1-99)")
+	var duration = flag.String("duration", "15s", "specify a length for the test run. Append 'm', 'h', or 'd' to specify minutes, hours or days.")
```

Comment:
you can use flag.Duration here

Reply:
Basically, all the flags should eventually be converted into strings, but I agree that this emphasis the purpose of each variable a little bit.

Reply:
Ah, did not notice. Then I guess it's fine and it's fine to rely on the cyclictestBinary to verify their correctness

Reply:
My mistake here, it must be a flag because duration can't support more than hours, and we need days as well